### PR TITLE
Add .NET Framework dotnet-isolated template

### PR DESCRIPTION
**Pull request by liliankasem**
Addresses this issue:

- https://github.com/Azure/azure-functions-templates/issues/1229

Tested locally, the templates are created as expected

### CODE REVIEW

File: Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/NetFx48/Company.FunctionApp.csproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <TargetFramework>net48</TargetFramework>
+    <AzureFunctionsVersion>AzureFunctionsVersionValue</AzureFunctionsVersion>
+    <OutputType>Exe</OutputType>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\key.snk</AssemblyOriginatorKeyFile>
```
Comment (fabiocav) [inline Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/NetFx48/Company.FunctionApp.csproj:8]: The templates wouldn't have signing by default.

File: Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/NetFx48/Program.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Azure.Functions.Worker;
+
+FunctionsDebugger.Enable();
```
Comment (fabiocav) [inline Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/NetFx48/Program.cs:6]: 👍🏼  This is the key thing I wanted to make sure wasn't missed :)

**Review by soninaren — COMMENTED** [review]
Left a single comment, we can revise the nuspec file post testing.

File: Build/PackageFiles/Dotnet_precompiled/ProjectTemplates-Isolated_v4.x.nuspec [hunk 3]
```diff
@@ -16,13 +16,20 @@
...
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Net6/Company.FunctionApp.csproj" target="content/ProjectTemplate-CSharp/Net6/Company.FunctionApp.csproj" />
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Net6/Program.cs" target="content/ProjectTemplate-CSharp/Net6/Program.cs" />
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Net6/host.json" target="content/ProjectTemplate-CSharp/Net6/host.json" />
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Net6/local.settings.json" target="content/ProjectTemplate-CSharp/Net6/local.settings.json" />
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Net6/Properties/launchSettings.json" target="content/ProjectTemplate-CSharp/Net6/Properties/launchSettings.json" />
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Net6/.template.config/template.json" target="content/ProjectTemplate-CSharp/Net6/.template.config/template.json" />
```
[the remark this answers is not in the document]
Comment (liliankasem) [reply Build/PackageFiles/Dotnet_precompiled/ProjectTemplates-Isolated_v4.x.nuspec:25]: addressed offline

**Review by soninaren — APPROVED** [review]

**Pull request merged**